packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -732,6 +732,10 @@ export class IndexedDbPersistence implements Persistence {
     );
   }
 
+  static async clearPersistence(dbName: string): Promise<void> {
+    await SimpleDb.delete(dbName);
```

Comment:
In this case, I would just `return SimpleDb.delete(dbName)`. This function will also no longer need to be marked `async`.